src/excmds.ts

```diff
@@ -383,7 +383,11 @@ export async function guiset_quiet(rule: string, option: string) {
     await Native.write(profile_dir + "/chrome/userChrome.css.tri.bak", cssstr)
 
     // Modify and write new CSS
-    const stylesheet = CSS.parse(cssstr)
+    const stylesheet = CSS.parse(cssstr, { silent: true })
+    if (stylesheet.stylesheet.parsingErrors.length) {
+        const error = stylesheet.stylesheet.parsingErrors[0]
+        throw new Error(`userChrome.css is malformed and cannot be parsed: ${error.reason} at ${error.line}:${error.column}. Please manually fix or delete it, then try again.`)
```

Comment:
```suggestion
        throw new Error(`Your current userChrome.css is malformed: ${error.reason} at ${error.line}:${error.column}. Fix or delete it and try again.`)
```

Better to keep these messages short so they can be displayed without scrolling